src/status_im/contexts/wallet/send/input_amount/component_spec.cljs

```diff
@@ -100,60 +100,4 @@
           (.then (fn []
                    (h/is-truthy (h/get-by-label-text :button-one))
                    (h/fire-event :press (h/get-by-label-text :button-one))
-                   (h/was-called on-confirm))))))
```

Comment:
why are we deleting all these tests? 🤔

Reply:
Because the tests were covering the input of more than limit which were reset to the limit. Now, when entering more than limit, nothing should happen except for changing the style.